app/helpers/view_helper.rb

```diff
@@ -4,8 +4,8 @@ def belongs_to_current_user?(object)
   end
 
   def collection_image?(coll)
-    if coll.approved_observations.last
-      if coll.approved_observations.last.image_file_name.present?
+    if coll.approved_observations.first
```

Comment:
Note: because the controller is delivering the collections in descending order, we actually need to pull the first one from the collection, since the first one is the most recently updated one.